CHANGELOG.md

```diff
@@ -1,3 +1,439 @@
+## Parity [v1.7.0](https://github.com/paritytech/parity/releases/tag/v1.7.0) (2017-07-17)
+
+[WIP] @TODO Random list of new features
+
+- Light client support
```

Comment:
would like to clarify experimental here. it will likely be a few releases before it feels mature